modules/indexing/src/main/java/org/apache/ignite/internal/processors/query/h2/H2TableDescriptor.java

```diff
@@ -325,7 +325,7 @@ GridLuceneIndex luceneIndex() {
     @NotNull private List<IndexColumn> extractKeyColumns(GridH2Table tbl, IndexColumn keyCol, IndexColumn affCol) {
         ArrayList<IndexColumn> keyCols;
 
-        if (isSql) {
```

Comment:
Hi! `H2TableDescriptor#sql()` also is used in `H2Utils#unwrapKeyColumns` for indexes information. Should we change logic there too?

Reply:
I see this change, do we need `H2TableDescriptor#sql()` method then at all?

Reply:
>  do we need H2TableDescriptor#sql() method then at all?
I guess, yes. Because we have to support compatible behavior.
May it will be dropped in the future.

Reply:
But `H2TableDescriptor#sql()` is never used now and is in internal package. Is it used somehow for compatibility?